frontend/src/metabase/dashboard/components/DashCard/Dashcard.unit.spec.tsx

```diff
@@ -138,4 +139,42 @@ describe("DashCard", () => {
       opacity: 0.25,
     });
   });
+
+  it("shows a link visualization", () => {
+    const linkCard = createMockLinkDashboardCard({
+      url: "https://xkcd.com/327",
+    });
+    const board = {
+      ...dashboard,
+      ordered_cards: [linkCard],
+    };
+    setup({
+      dashboard: board,
+      dashcard: linkCard,
+      dashcardData: {},
+    });
+    expect(screen.getByText("https://xkcd.com/327")).toBeVisible();
+  });
+
+  it("in parameter editing mode, shows faded link dashcard", () => {
+    const linkCard = createMockLinkDashboardCard({
+      url: "https://xkcd.com/327",
+    });
+    const board = {
+      ...dashboard,
+      ordered_cards: [linkCard],
+    };
+    setup({
+      dashboard: board,
+      dashcard: linkCard,
+      dashcardData: {},
+      isEditing: true,
+      isEditingParameter: true,
+    });
+
+    expect(screen.getByText("https://xkcd.com/327")).toBeVisible();
+    expect(screen.getByTestId("custom-view-text-link")).toHaveStyle({
```

Comment:
@alxnddr Any ideas on what to do here? We don't normally check for styles, but this change is specifically to adjust opacity.

Reply:
I think it is good because we check styles here not to verify the looks but rather that we check the changed logical state of the widget